vagrant-settings.yaml

```diff
@@ -0,0 +1,45 @@
+# copy it to vagrant-settings.yaml then Vagrantfile
+# will use values from this file
+
+slaves_count: 2
+slaves_ram: 1024
+master_image: solar-master
```

Comment:
IT will require solar-master on local env :) We need solar-project/solar-master (preferably most recent version pre 0.3.0)